app/redux/network/reducer.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import type { Action } from '/types';
+import { CHECK_NETWORK_CONNECTION } from './actions';
+
+const initialState = {
+  isNetworkConnected: true
```

Comment:
isConnected is more appropriate name for this

Reply:
changing